factory-helper.gemspec

```diff
@@ -0,0 +1,26 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'factory-helper/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "factory-helper"
+  spec.version       = FactoryHelper::VERSION
+  spec.authors       = ["theB264"]
+  spec.email         = ["public88878878887@hotmail.com"]
+  spec.summary       = %q{Easily generate fake data}
+  spec.description   = %q{Easily generate fake data: names, addresses, phone numbers, etc.}
+  spec.homepage      = "https://github.com/factory-helper/factory-helper"
+  spec.license       = "MIT"
+  spec.required_ruby_version = ">= 1.9.3"
```

Comment:
lib/extensions/array.rb implements just that, so we _could_ support 1.8.7. I'd prefer not to, given this version's age and dropped official support, and to get rid of the needless monkey patch.